### ISSUE TITLE
GO:0140582 not recognized

### ISSUE THREAD

**Issue by pfey03** [geneontology/noctua-form#136]
GO:0140582: adenylate cyclase-activating G protein-coupled cAMP receptor signaling pathway 
which has been around for a while does not come up in form or editing tool, where I tried to change a parent term to this specific term. ID or term were not recognized.

In graph editor I could edit it quite simply, but wanted to test the editor feature.

**Comment by ukemi** [comment]
Hi @pfey03. I saw this yesterday. For now, if you use the identifier, GO:0140582, you will see it.

**Comment by vanaukenk** [comment]
@pfey03 
I can see this in the noctua form as a BP term.  Is that where you tried the autocomplete?

**Comment by pfey03** [comment]
@vanaukenk and @ukemi maybe because I use Safari for all? I tried in ART with ID and term and it showed me weird dopamine terms but not the one I need. I then went to graph and edited it there and it worked.

Also, when you import annotations and the gene has a lot often the term you want is not there. That's why I had another term and then wanted to edit.

**Comment by vanaukenk** [comment]
Thankls @pfey03 

It would be _really_ helpful to share a screen with you to see the exact behavior and determine if there's a fix we need to implement.

I'd also like to make sure I understand your second comment about importing annotations.

Are you available for a quick conference call some time?

**Comment by pfey03** [comment]
@vanaukenk Sure, but maybe next week as the University gave us off from tomorrow through Monday for a long holiday weekend. You have more meetings than me I have two Thursday mornings and the GO call Tuesdays regularly, otherwise relatively flexible.

Also in Safari you cannot scroll the list of terms when you import annotations in the form. Pascale told me it's limited to 10 terms anyway?

**Comment by pfey03** [comment]
![Boden_Eldon_Cotton_Cardi(S)](https://user-images.githubusercontent.com/1349602/124015528-678e9f80-d9aa-11eb-83d9-5055ecdc3b08.png)

here is a screenshot when entering the GO ID into ART

**Comment by pfey03** [comment]
![ART](https://user-images.githubusercontent.com/1349602/124015946-b89e9380-d9aa-11eb-915d-008c06f21bab.png)

This is what I see entering the GO ID in ART. I can scroll this window but the term doesn't show up.

**Comment by ukemi** [comment]
@pfey03 you are trying to retrieve a process from a field that is restricted to functions. You can't have a process enabled by a gene product.

**Comment by vanaukenk** [comment]
@pfey03

Each of the ontology autocomplete fields in the Noctua form and ART is restricted according to the correct relations between gps and GO terms, as well as between GO terms.

For example, you can only replace an MF with an MF.

Let's touch base next week to look at it together.

**Comment by pfey03** [comment]
Hmm no I did replace a function with function. Maybe when I tried to reproduce for snapshot, let me try again.

**Comment by pfey03** [comment]
I changed the process for carA-1 back to the term it was and tried again in ART to replace it with GO:0140582 and could not. And you are right, I go to the pen in the process field and it shows me functions. Very strange! 

The pen is in lower right of the field but I'm in the process field, the pen doesn't do the right thing it seems.

**Comment by pfey03** [comment]
I leave it now and show you next week

**Comment by pfey03** [comment]
![process as MF in form](https://user-images.githubusercontent.com/1349602/124042626-dc73d080-d9ce-11eb-82d6-5a975887efaf.png)

see process GO:0007189 in form as MF

**Comment by pfey03** [comment]
added term in graph editor

**Comment by pfey03** [comment]
ok, I could add the process directly to annoton in graph and that caused the confusion. Now changed it and added function and then process part_of. Then editing worked. 

Oh well, I always manage to find ways to break things as I sometimes try short cuts and like to find how to break things. But that's probably just an issue of graph editor that is still too loose. After I saw the process in function field I noticed the editor gave me too much freedom to toy around.

Happy holiday!
